components/centraldashboard/public/components/registration-page.css

```diff
@@ -15,7 +15,7 @@
 }
 h1,h2,h3 {margin: 0;font-family: Google Sans;color: var(--heading-color);font-weight: 400}
```

Comment:
This is tough to read/maintain all being crammed onto the same lines. Can you run beautifier over this file?

Reply:
Done!